### PR TITLE
 Django Tutorial Part 7: Sessions framework

### PR DESCRIPTION
**Pull request by sonnguyen28**

### CODE REVIEW

**Comment by sonnguyen28** [comment]
Ready

File: locallibrary/catalog/templates/index.html [hunk 1]
```diff
@@ -11,4 +11,6 @@ <h2>{% trans "Dynamic content" %}</h2>
     <li><strong>{% trans "Copies available:" %}</strong> {{ num_instances_available }}</li>
     <li><strong>{% trans "Authors:" %}</strong> {{ num_authors }}</li>
   </ul>
+
+  <p>You have visited this page {{ num_visits }} time{{ num_visits|pluralize }}.</p>
```
Comment (huongnt-2545) [inline locallibrary/catalog/templates/index.html]: translate text

**Comment by sonnguyen28** [comment]
Ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**